routers/web/repo/projects.go

```diff
@@ -383,16 +383,41 @@ func UpdateIssueProject(ctx *context.Context) {
 		ctx.ServerError("LoadProjects", err)
 		return
 	}
+	if _, err := issues.LoadRepositories(ctx); err != nil {
+		ctx.ServerError("LoadProjects", err)
+		return
+	}
 
 	projectID := ctx.FormInt64("id")
+	var dstColumnID int64
+	if projectID > 0 {
+		dstProject, err := project_model.GetProjectByID(ctx, projectID)
+		if err != nil {
+			ctx.ServerError("GetProjectByID", err)
```

Comment:
A similar problem to #30580.
Didn't look into it, but tested if return `ServerError`, you will see nothing in frontend.

Reply:
https://github.com/go-gitea/gitea/pull/29722/files#diff-4f89ee029f289a1c40cf85fa40b0a10046a93d5fe1dac8d158e98bebd2e6e49eR83
the response will not be handled, so no matter what error occurred user will not see any error message if the operation is failed.

Reply:
81d455fa6cd22994ee99eb819f13e2b10fc2e874

Reply:
I think we can ignore this issue in this PR, and fix it later.